tests/helm/test_git_sync_scheduler.py

```diff
@@ -0,0 +1,136 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+import unittest
+
+import yaml
+from kubernetes.client import models as k8s
+
+from airflow.utils.helm_template_generator import render_chart, render_k8s_object
+
+OBJECT_COUNT_IN_BASIC_DEPLOYMENT = 22
+
+
+class TestGitSyncScheduler(unittest.TestCase):
+    def setUp(self) -> None:
+        with open(sys.path[0] + "/tests/helm/test_templates/"
```

Comment:
I don't see any benefit in keeping this as separate files. It just causes us to switch and look for the related file.

Reply:
Do you mean separate files for different test scenarios or separate files for different yamls?

Reply:
Separate files at all. Parameters can be passed as a dictionary in Python files.